### ISSUE TITLE
Null safety migration

### ISSUE THREAD

**Issue by dna-f** [gurleensethi/sailor#58]
Hi, is Null safety migration on roadmap?
Thanks

**Comment by mendoxe** [comment]
Hi, I would really appreciate more info about migrating to null safety, thanks!

**Comment by yagoliveira92** [comment]
I create a repository for a port to Sailor in Null Safety:

https://github.com/yagoliveira92/seafarer

I need fix the test to publish in the pub.dev. PR are welcome.

**Comment by yagoliveira92** [comment]
> I create a repository for a port to Sailor in Null Safety:
> 
> https://github.com/yagoliveira92/seafarer
> 
> I need fix the test to publish in the pub.dev. PR are welcome.

Package created, can access on https://pub.dev/packages/seafarer

**Comment by danilocubos** [comment]
Is this package still under development?

**Comment by yagoliveira92** [comment]
Yes, I have plans to upgrade this package. Actuality I using in productions
projects.

Em sex, 2 de jul de 2021 19:41, danilocubos ***@***.***>
escreveu:

**Comment by danilocubos** [comment]
Hi @yagoliveira92! Are you talking about Sailor or Seafarer? 

> Yes, I have plans to upgrade this package. Actuality I using in productions projects.

**Comment by yagoliveira92** [comment]
Hi! I'm use Seafarer because Null Safety.

Em dom, 4 de jul de 2021 21:08, danilocubos ***@***.***>
escreveu:

**Comment by officialismailshah** [comment]
Null safety is must in dart

sailor package with null safety with [sailor](https://github.com/proismailshah/sailor.git)

or you can use the "seafarer" package that is exact replica of this package on pub dev
